docs/guides/integration/open-from-github.md

```diff
@@ -157,6 +157,7 @@ Either `awesome-lib.new` or `awesome-lib.com/new` are both quite handy ways to h
 
 Here are some examples of how others are doing it:
 
+- [astro.new](https://astro.new/); eg: [astro.new/blog?on=stackblitz](https://astro.new/blog?on=stackblitz)
```

Comment:
In our docs we are trying to avoid shortcuts in favor of plain English alternatives. Would you be comfortable with this change?

```suggestion
- [astro.new](https://astro.new/); for example: [astro.new/blog?on=stackblitz](https://astro.new/blog?on=stackblitz)
```